src/main/java/com/fasterxml/jackson/databind/ser/BeanPropertyWriter.java

```diff
@@ -325,7 +326,12 @@ protected BeanPropertyWriter(BeanPropertyWriter base, PropertyName name) {
                     base._internalSettings);
         }
         _cfgSerializationType = base._cfgSerializationType;
-        _dynamicSerializers = base._dynamicSerializers;
+        if(base instanceof UnwrappingBeanPropertyWriter) {
```

Comment:
Dependency from super class to subclasses doesn't seem to be the right direction.

Reply:
Yes, did the same thing in a better way now